[2022-06-17 05:03:41,587] ERROR Closing socket for 127.0.0.1:9092-127.0.0.1:50476-4 because of error (kafka.network.Processor)
org.apache.kafka.common.errors.InvalidRequestException: Error parsing request header. Our best guess of the apiKey is: 0
Caused by: java.nio.BufferUnderflowException
	at java.base/java.nio.Buffer.nextGetIndex(Buffer.java:651)
	at java.base/java.nio.HeapByteBuffer.getShort(HeapByteBuffer.java:350)
	at org.apache.kafka.common.protocol.ByteBufferAccessor.readShort(ByteBufferAccessor.java:38)
	at org.apache.kafka.common.message.RequestHeaderData.read(RequestHeaderData.java:119)
	at org.apache.kafka.common.message.RequestHeaderData.<init>(RequestHeaderData.java:83)
	at org.apache.kafka.common.requests.RequestHeader.parse(RequestHeader.java:93)
	at kafka.network.Processor.$anonfun$processCompletedReceives$1(SocketServer.scala:875)
	at kafka.network.Processor.$anonfun$processCompletedReceives$1$adapted(SocketServer.scala:871)
	at scala.collection.Iterator.foreach(Iterator.scala:941)
	at scala.collection.Iterator.foreach$(Iterator.scala:941)
	at scala.collection.AbstractIterator.foreach(Iterator.scala:1429)
	at scala.collection.IterableLike.foreach(IterableLike.scala:74)
	at scala.collection.IterableLike.foreach$(IterableLike.scala:73)
	at scala.collection.AbstractIterable.foreach(Iterable.scala:56)
	at kafka.network.Processor.processCompletedReceives(SocketServer.scala:871)
	at kafka.network.Processor.run(SocketServer.scala:761)
	at java.base/java.lang.Thread.run(Thread.java:829)
[2022-06-17 05:03:41,590] ERROR Exception while processing request from 127.0.0.1:9092-127.0.0.1:50476-4 (kafka.network.Processor)
org.apache.kafka.common.errors.InvalidRequestException: Error parsing request header. Our best guess of the apiKey is: 0
Caused by: java.nio.BufferUnderflowException
	at java.base/java.nio.Buffer.nextGetIndex(Buffer.java:651)
	at java.base/java.nio.HeapByteBuffer.getShort(HeapByteBuffer.java:350)
	at org.apache.kafka.common.protocol.ByteBufferAccessor.readShort(ByteBufferAccessor.java:38)
	at org.apache.kafka.common.message.RequestHeaderData.read(RequestHeaderData.java:119)
	at org.apache.kafka.common.message.RequestHeaderData.<init>(RequestHeaderData.java:83)
	at org.apache.kafka.common.requests.RequestHeader.parse(RequestHeader.java:93)
	at kafka.network.Processor.$anonfun$processCompletedReceives$1(SocketServer.scala:875)
	at kafka.network.Processor.$anonfun$processCompletedReceives$1$adapted(SocketServer.scala:871)
	at scala.collection.Iterator.foreach(Iterator.scala:941)
	at scala.collection.Iterator.foreach$(Iterator.scala:941)
	at scala.collection.AbstractIterator.foreach(Iterator.scala:1429)
	at scala.collection.IterableLike.foreach(IterableLike.scala:74)
	at scala.collection.IterableLike.foreach$(IterableLike.scala:73)
	at scala.collection.AbstractIterable.foreach(Iterable.scala:56)
	at kafka.network.Processor.processCompletedReceives(SocketServer.scala:871)
	at kafka.network.Processor.run(SocketServer.scala:761)
	at java.base/java.lang.Thread.run(Thread.java:829)
[2022-06-17 05:03:42,498] ERROR Closing socket for 127.0.0.1:9092-127.0.0.1:50478-5 because of error (kafka.network.Processor)
org.apache.kafka.common.errors.InvalidRequestException: Error parsing request header. Our best guess of the apiKey is: 0
Caused by: java.nio.BufferUnderflowException
	at java.base/java.nio.Buffer.nextGetIndex(Buffer.java:651)
	at java.base/java.nio.HeapByteBuffer.getShort(HeapByteBuffer.java:350)
	at org.apache.kafka.common.protocol.ByteBufferAccessor.readShort(ByteBufferAccessor.java:38)
	at org.apache.kafka.common.message.RequestHeaderData.read(RequestHeaderData.java:119)
	at org.apache.kafka.common.message.RequestHeaderData.<init>(RequestHeaderData.java:83)
	at org.apache.kafka.common.requests.RequestHeader.parse(RequestHeader.java:93)
	at kafka.network.Processor.$anonfun$processCompletedReceives$1(SocketServer.scala:875)
	at kafka.network.Processor.$anonfun$processCompletedReceives$1$adapted(SocketServer.scala:871)
	at scala.collection.Iterator.foreach(Iterator.scala:941)
	at scala.collection.Iterator.foreach$(Iterator.scala:941)
	at scala.collection.AbstractIterator.foreach(Iterator.scala:1429)
	at scala.collection.IterableLike.foreach(IterableLike.scala:74)
	at scala.collection.IterableLike.foreach$(IterableLike.scala:73)
	at scala.collection.AbstractIterable.foreach(Iterable.scala:56)
	at kafka.network.Processor.processCompletedReceives(SocketServer.scala:871)
	at kafka.network.Processor.run(SocketServer.scala:761)
	at java.base/java.lang.Thread.run(Thread.java:829)
[2022-06-17 05:03:42,499] ERROR Exception while processing request from 127.0.0.1:9092-127.0.0.1:50478-5 (kafka.network.Processor)
org.apache.kafka.common.errors.InvalidRequestException: Error parsing request header. Our best guess of the apiKey is: 0
Caused by: java.nio.BufferUnderflowException
	at java.base/java.nio.Buffer.nextGetIndex(Buffer.java:651)
	at java.base/java.nio.HeapByteBuffer.getShort(HeapByteBuffer.java:350)
	at org.apache.kafka.common.protocol.ByteBufferAccessor.readShort(ByteBufferAccessor.java:38)
	at org.apache.kafka.common.message.RequestHeaderData.read(RequestHeaderData.java:119)
	at org.apache.kafka.common.message.RequestHeaderData.<init>(RequestHeaderData.java:83)
	at org.apache.kafka.common.requests.RequestHeader.parse(RequestHeader.java:93)
	at kafka.network.Processor.$anonfun$processCompletedReceives$1(SocketServer.scala:875)
	at kafka.network.Processor.$anonfun$processCompletedReceives$1$adapted(SocketServer.scala:871)
	at scala.collection.Iterator.foreach(Iterator.scala:941)
	at scala.collection.Iterator.foreach$(Iterator.scala:941)
	at scala.collection.AbstractIterator.foreach(Iterator.scala:1429)
	at scala.collection.IterableLike.foreach(IterableLike.scala:74)
	at scala.collection.IterableLike.foreach$(IterableLike.scala:73)
	at scala.collection.AbstractIterable.foreach(Iterable.scala:56)
	at kafka.network.Processor.processCompletedReceives(SocketServer.scala:871)
	at kafka.network.Processor.run(SocketServer.scala:761)
	at java.base/java.lang.Thread.run(Thread.java:829)
[2022-06-17 05:03:43,362] ERROR Closing socket for 127.0.0.1:9092-127.0.0.1:50480-5 because of error (kafka.network.Processor)
org.apache.kafka.common.errors.InvalidRequestException: Error parsing request header. Our best guess of the apiKey is: 0
Caused by: java.nio.BufferUnderflowException
	at java.base/java.nio.Buffer.nextGetIndex(Buffer.java:651)
	at java.base/java.nio.HeapByteBuffer.getShort(HeapByteBuffer.java:350)
	at org.apache.kafka.common.protocol.ByteBufferAccessor.readShort(ByteBufferAccessor.java:38)
	at org.apache.kafka.common.message.RequestHeaderData.read(RequestHeaderData.java:119)
	at org.apache.kafka.common.message.RequestHeaderData.<init>(RequestHeaderData.java:83)
	at org.apache.kafka.common.requests.RequestHeader.parse(RequestHeader.java:93)
	at kafka.network.Processor.$anonfun$processCompletedReceives$1(SocketServer.scala:875)
	at kafka.network.Processor.$anonfun$processCompletedReceives$1$adapted(SocketServer.scala:871)
	at scala.collection.Iterator.foreach(Iterator.scala:941)
	at scala.collection.Iterator.foreach$(Iterator.scala:941)
	at scala.collection.AbstractIterator.foreach(Iterator.scala:1429)
	at scala.collection.IterableLike.foreach(IterableLike.scala:74)
	at scala.collection.IterableLike.foreach$(IterableLike.scala:73)
	at scala.collection.AbstractIterable.foreach(Iterable.scala:56)
	at kafka.network.Processor.processCompletedReceives(SocketServer.scala:871)
	at kafka.network.Processor.run(SocketServer.scala:761)
	at java.base/java.lang.Thread.run(Thread.java:829)
[2022-06-17 05:03:43,363] ERROR Exception while processing request from 127.0.0.1:9092-127.0.0.1:50480-5 (kafka.network.Processor)
org.apache.kafka.common.errors.InvalidRequestException: Error parsing request header. Our best guess of the apiKey is: 0
Caused by: java.nio.BufferUnderflowException
	at java.base/java.nio.Buffer.nextGetIndex(Buffer.java:651)
	at java.base/java.nio.HeapByteBuffer.getShort(HeapByteBuffer.java:350)
	at org.apache.kafka.common.protocol.ByteBufferAccessor.readShort(ByteBufferAccessor.java:38)
	at org.apache.kafka.common.message.RequestHeaderData.read(RequestHeaderData.java:119)
	at org.apache.kafka.common.message.RequestHeaderData.<init>(RequestHeaderData.java:83)
	at org.apache.kafka.common.requests.RequestHeader.parse(RequestHeader.java:93)
	at kafka.network.Processor.$anonfun$processCompletedReceives$1(SocketServer.scala:875)
	at kafka.network.Processor.$anonfun$processCompletedReceives$1$adapted(SocketServer.scala:871)
	at scala.collection.Iterator.foreach(Iterator.scala:941)
	at scala.collection.Iterator.foreach$(Iterator.scala:941)
	at scala.collection.AbstractIterator.foreach(Iterator.scala:1429)
	at scala.collection.IterableLike.foreach(IterableLike.scala:74)
	at scala.collection.IterableLike.foreach$(IterableLike.scala:73)
	at scala.collection.AbstractIterable.foreach(Iterable.scala:56)
	at kafka.network.Processor.processCompletedReceives(SocketServer.scala:871)
	at kafka.network.Processor.run(SocketServer.scala:761)
	at java.base/java.lang.Thread.run(Thread.java:829)
[2022-06-17 05:03:54,720] ERROR Closing socket for 127.0.0.1:9092-127.0.0.1:50482-5 because of error (kafka.network.Processor)
org.apache.kafka.common.errors.InvalidRequestException: Error parsing request header. Our best guess of the apiKey is: 0
Caused by: java.nio.BufferUnderflowException
	at java.base/java.nio.Buffer.nextGetIndex(Buffer.java:651)
	at java.base/java.nio.HeapByteBuffer.getShort(HeapByteBuffer.java:350)
	at org.apache.kafka.common.protocol.ByteBufferAccessor.readShort(ByteBufferAccessor.java:38)
	at org.apache.kafka.common.message.RequestHeaderData.read(RequestHeaderData.java:119)
	at org.apache.kafka.common.message.RequestHeaderData.<init>(RequestHeaderData.java:83)
	at org.apache.kafka.common.requests.RequestHeader.parse(RequestHeader.java:93)
	at kafka.network.Processor.$anonfun$processCompletedReceives$1(SocketServer.scala:875)
	at kafka.network.Processor.$anonfun$processCompletedReceives$1$adapted(SocketServer.scala:871)
	at scala.collection.Iterator.foreach(Iterator.scala:941)
	at scala.collection.Iterator.foreach$(Iterator.scala:941)
	at scala.collection.AbstractIterator.foreach(Iterator.scala:1429)
	at scala.collection.IterableLike.foreach(IterableLike.scala:74)
	at scala.collection.IterableLike.foreach$(IterableLike.scala:73)
	at scala.collection.AbstractIterable.foreach(Iterable.scala:56)
	at kafka.network.Processor.processCompletedReceives(SocketServer.scala:871)
	at kafka.network.Processor.run(SocketServer.scala:761)
	at java.base/java.lang.Thread.run(Thread.java:829)
[2022-06-17 05:03:54,720] ERROR Exception while processing request from 127.0.0.1:9092-127.0.0.1:50482-5 (kafka.network.Processor)
org.apache.kafka.common.errors.InvalidRequestException: Error parsing request header. Our best guess of the apiKey is: 0
Caused by: java.nio.BufferUnderflowException
	at java.base/java.nio.Buffer.nextGetIndex(Buffer.java:651)
	at java.base/java.nio.HeapByteBuffer.getShort(HeapByteBuffer.java:350)
	at org.apache.kafka.common.protocol.ByteBufferAccessor.readShort(ByteBufferAccessor.java:38)
	at org.apache.kafka.common.message.RequestHeaderData.read(RequestHeaderData.java:119)
	at org.apache.kafka.common.message.RequestHeaderData.<init>(RequestHeaderData.java:83)
	at org.apache.kafka.common.requests.RequestHeader.parse(RequestHeader.java:93)
	at kafka.network.Processor.$anonfun$processCompletedReceives$1(SocketServer.scala:875)
	at kafka.network.Processor.$anonfun$processCompletedReceives$1$adapted(SocketServer.scala:871)
	at scala.collection.Iterator.foreach(Iterator.scala:941)
	at scala.collection.Iterator.foreach$(Iterator.scala:941)
	at scala.collection.AbstractIterator.foreach(Iterator.scala:1429)
	at scala.collection.IterableLike.foreach(IterableLike.scala:74)
	at scala.collection.IterableLike.foreach$(IterableLike.scala:73)
	at scala.collection.AbstractIterable.foreach(Iterable.scala:56)
	at kafka.network.Processor.processCompletedReceives(SocketServer.scala:871)
	at kafka.network.Processor.run(SocketServer.scala:761)
	at java.base/java.lang.Thread.run(Thread.java:829)
[2022-06-17 05:03:57,059] ERROR Closing socket for 127.0.0.1:9092-127.0.0.1:50484-6 because of error (kafka.network.Processor)
org.apache.kafka.common.errors.InvalidRequestException: Error parsing request header. Our best guess of the apiKey is: 0
Caused by: java.nio.BufferUnderflowException
	at java.base/java.nio.Buffer.nextGetIndex(Buffer.java:651)
	at java.base/java.nio.HeapByteBuffer.getShort(HeapByteBuffer.java:350)
	at org.apache.kafka.common.protocol.ByteBufferAccessor.readShort(ByteBufferAccessor.java:38)
	at org.apache.kafka.common.message.RequestHeaderData.read(RequestHeaderData.java:119)
	at org.apache.kafka.common.message.RequestHeaderData.<init>(RequestHeaderData.java:83)
	at org.apache.kafka.common.requests.RequestHeader.parse(RequestHeader.java:93)
	at kafka.network.Processor.$anonfun$processCompletedReceives$1(SocketServer.scala:875)
	at kafka.network.Processor.$anonfun$processCompletedReceives$1$adapted(SocketServer.scala:871)
	at scala.collection.Iterator.foreach(Iterator.scala:941)
	at scala.collection.Iterator.foreach$(Iterator.scala:941)
	at scala.collection.AbstractIterator.foreach(Iterator.scala:1429)
	at scala.collection.IterableLike.foreach(IterableLike.scala:74)
	at scala.collection.IterableLike.foreach$(IterableLike.scala:73)
	at scala.collection.AbstractIterable.foreach(Iterable.scala:56)
	at kafka.network.Processor.processCompletedReceives(SocketServer.scala:871)
	at kafka.network.Processor.run(SocketServer.scala:761)
	at java.base/java.lang.Thread.run(Thread.java:829)
[2022-06-17 05:03:57,059] ERROR Exception while processing request from 127.0.0.1:9092-127.0.0.1:50484-6 (kafka.network.Processor)
org.apache.kafka.common.errors.InvalidRequestException: Error parsing request header. Our best guess of the apiKey is: 0
Caused by: java.nio.BufferUnderflowException
	at java.base/java.nio.Buffer.nextGetIndex(Buffer.java:651)
	at java.base/java.nio.HeapByteBuffer.getShort(HeapByteBuffer.java:350)
	at org.apache.kafka.common.protocol.ByteBufferAccessor.readShort(ByteBufferAccessor.java:38)
	at org.apache.kafka.common.message.RequestHeaderData.read(RequestHeaderData.java:119)
	at org.apache.kafka.common.message.RequestHeaderData.<init>(RequestHeaderData.java:83)
	at org.apache.kafka.common.requests.RequestHeader.parse(RequestHeader.java:93)
	at kafka.network.Processor.$anonfun$processCompletedReceives$1(SocketServer.scala:875)
	at kafka.network.Processor.$anonfun$processCompletedReceives$1$adapted(SocketServer.scala:871)
	at scala.collection.Iterator.foreach(Iterator.scala:941)
	at scala.collection.Iterator.foreach$(Iterator.scala:941)
	at scala.collection.AbstractIterator.foreach(Iterator.scala:1429)
	at scala.collection.IterableLike.foreach(IterableLike.scala:74)
	at scala.collection.IterableLike.foreach$(IterableLike.scala:73)
	at scala.collection.AbstractIterable.foreach(Iterable.scala:56)
	at kafka.network.Processor.processCompletedReceives(SocketServer.scala:871)
	at kafka.network.Processor.run(SocketServer.scala:761)
	at java.base/java.lang.Thread.run(Thread.java:829)
[2022-06-17 05:03:57,650] ERROR Closing socket for 127.0.0.1:9092-127.0.0.1:50486-6 because of error (kafka.network.Processor)
org.apache.kafka.common.errors.InvalidRequestException: Error parsing request header. Our best guess of the apiKey is: 0
Caused by: java.nio.BufferUnderflowException
	at java.base/java.nio.Buffer.nextGetIndex(Buffer.java:651)
	at java.base/java.nio.HeapByteBuffer.getShort(HeapByteBuffer.java:350)
	at org.apache.kafka.common.protocol.ByteBufferAccessor.readShort(ByteBufferAccessor.java:38)
	at org.apache.kafka.common.message.RequestHeaderData.read(RequestHeaderData.java:119)
	at org.apache.kafka.common.message.RequestHeaderData.<init>(RequestHeaderData.java:83)
	at org.apache.kafka.common.requests.RequestHeader.parse(RequestHeader.java:93)
	at kafka.network.Processor.$anonfun$processCompletedReceives$1(SocketServer.scala:875)
	at kafka.network.Processor.$anonfun$processCompletedReceives$1$adapted(SocketServer.scala:871)
	at scala.collection.Iterator.foreach(Iterator.scala:941)
	at scala.collection.Iterator.foreach$(Iterator.scala:941)
	at scala.collection.AbstractIterator.foreach(Iterator.scala:1429)
	at scala.collection.IterableLike.foreach(IterableLike.scala:74)
	at scala.collection.IterableLike.foreach$(IterableLike.scala:73)
	at scala.collection.AbstractIterable.foreach(Iterable.scala:56)
	at kafka.network.Processor.processCompletedReceives(SocketServer.scala:871)
	at kafka.network.Processor.run(SocketServer.scala:761)
	at java.base/java.lang.Thread.run(Thread.java:829)
[2022-06-17 05:03:57,650] ERROR Exception while processing request from 127.0.0.1:9092-127.0.0.1:50486-6 (kafka.network.Processor)
org.apache.kafka.common.errors.InvalidRequestException: Error parsing request header. Our best guess of the apiKey is: 0
Caused by: java.nio.BufferUnderflowException
	at java.base/java.nio.Buffer.nextGetIndex(Buffer.java:651)
	at java.base/java.nio.HeapByteBuffer.getShort(HeapByteBuffer.java:350)
	at org.apache.kafka.common.protocol.ByteBufferAccessor.readShort(ByteBufferAccessor.java:38)
	at org.apache.kafka.common.message.RequestHeaderData.read(RequestHeaderData.java:119)
	at org.apache.kafka.common.message.RequestHeaderData.<init>(RequestHeaderData.java:83)
	at org.apache.kafka.common.requests.RequestHeader.parse(RequestHeader.java:93)
	at kafka.network.Processor.$anonfun$processCompletedReceives$1(SocketServer.scala:875)
	at kafka.network.Processor.$anonfun$processCompletedReceives$1$adapted(SocketServer.scala:871)
	at scala.collection.Iterator.foreach(Iterator.scala:941)
	at scala.collection.Iterator.foreach$(Iterator.scala:941)
	at scala.collection.AbstractIterator.foreach(Iterator.scala:1429)
	at scala.collection.IterableLike.foreach(IterableLike.scala:74)
	at scala.collection.IterableLike.foreach$(IterableLike.scala:73)
	at scala.collection.AbstractIterable.foreach(Iterable.scala:56)
	at kafka.network.Processor.processCompletedReceives(SocketServer.scala:871)
	at kafka.network.Processor.run(SocketServer.scala:761)
	at java.base/java.lang.Thread.run(Thread.java:829)
[2022-06-17 05:09:03,870] INFO [GroupMetadataManager brokerId=0] Removed 0 expired offsets in 4 milliseconds. (kafka.coordinator.group.GroupMetadataManager)
[2022-06-17 05:17:07,712] INFO [GroupCoordinator 0]: Member[group.instance.id None, member.id consumer-DepositTeam-2-2129f97a-4210-45a5-ae41-a924db5063c1] in group DepositTeam has left, removing it from the group (kafka.coordinator.group.GroupCoordinator)
[2022-06-17 05:17:07,712] INFO [GroupCoordinator 0]: Preparing to rebalance group DepositTeam in state PreparingRebalance with old generation 1 (__consumer_offsets-45) (reason: removing member consumer-DepositTeam-2-2129f97a-4210-45a5-ae41-a924db5063c1 on LeaveGroup) (kafka.coordinator.group.GroupCoordinator)
[2022-06-17 05:17:07,713] INFO [GroupCoordinator 0]: Group DepositTeam with generation 2 is now empty (__consumer_offsets-45) (kafka.coordinator.group.GroupCoordinator)
[2022-06-17 05:17:44,230] INFO [GroupCoordinator 0]: Preparing to rebalance group DepositTeam in state PreparingRebalance with old generation 2 (__consumer_offsets-45) (reason: Adding new member consumer-DepositTeam-2-0ecd7a34-1da2-4c66-9b31-2993bbd2e97a with group instance id None) (kafka.coordinator.group.GroupCoordinator)
[2022-06-17 05:17:44,231] INFO [GroupCoordinator 0]: Stabilized group DepositTeam generation 3 (__consumer_offsets-45) (kafka.coordinator.group.GroupCoordinator)
[2022-06-17 05:17:44,242] INFO [GroupCoordinator 0]: Assignment received from leader for group DepositTeam for generation 3 (kafka.coordinator.group.GroupCoordinator)
[2022-06-17 05:19:03,867] INFO [GroupMetadataManager brokerId=0] Removed 0 expired offsets in 0 milliseconds. (kafka.coordinator.group.GroupMetadataManager)
[2022-06-17 05:19:58,245] ERROR Closing socket for 127.0.0.1:9092-127.0.0.1:50524-11 because of error (kafka.network.Processor)
org.apache.kafka.common.errors.InvalidRequestException: Error parsing request header. Our best guess of the apiKey is: 0
Caused by: java.nio.BufferUnderflowException
	at java.base/java.nio.Buffer.nextGetIndex(Buffer.java:651)
	at java.base/java.nio.HeapByteBuffer.getShort(HeapByteBuffer.java:350)
	at org.apache.kafka.common.protocol.ByteBufferAccessor.readShort(ByteBufferAccessor.java:38)
	at org.apache.kafka.common.message.RequestHeaderData.read(RequestHeaderData.java:119)
	at org.apache.kafka.common.message.RequestHeaderData.<init>(RequestHeaderData.java:83)
	at org.apache.kafka.common.requests.RequestHeader.parse(RequestHeader.java:93)
	at kafka.network.Processor.$anonfun$processCompletedReceives$1(SocketServer.scala:875)
	at kafka.network.Processor.$anonfun$processCompletedReceives$1$adapted(SocketServer.scala:871)
	at scala.collection.Iterator.foreach(Iterator.scala:941)
	at scala.collection.Iterator.foreach$(Iterator.scala:941)
	at scala.collection.AbstractIterator.foreach(Iterator.scala:1429)
	at scala.collection.IterableLike.foreach(IterableLike.scala:74)
	at scala.collection.IterableLike.foreach$(IterableLike.scala:73)
	at scala.collection.AbstractIterable.foreach(Iterable.scala:56)
	at kafka.network.Processor.processCompletedReceives(SocketServer.scala:871)
	at kafka.network.Processor.run(SocketServer.scala:761)
	at java.base/java.lang.Thread.run(Thread.java:829)
[2022-06-17 05:19:58,245] ERROR Exception while processing request from 127.0.0.1:9092-127.0.0.1:50524-11 (kafka.network.Processor)
org.apache.kafka.common.errors.InvalidRequestException: Error parsing request header. Our best guess of the apiKey is: 0
Caused by: java.nio.BufferUnderflowException
	at java.base/java.nio.Buffer.nextGetIndex(Buffer.java:651)
	at java.base/java.nio.HeapByteBuffer.getShort(HeapByteBuffer.java:350)
	at org.apache.kafka.common.protocol.ByteBufferAccessor.readShort(ByteBufferAccessor.java:38)
	at org.apache.kafka.common.message.RequestHeaderData.read(RequestHeaderData.java:119)
	at org.apache.kafka.common.message.RequestHeaderData.<init>(RequestHeaderData.java:83)
	at org.apache.kafka.common.requests.RequestHeader.parse(RequestHeader.java:93)
	at kafka.network.Processor.$anonfun$processCompletedReceives$1(SocketServer.scala:875)
	at kafka.network.Processor.$anonfun$processCompletedReceives$1$adapted(SocketServer.scala:871)
	at scala.collection.Iterator.foreach(Iterator.scala:941)
	at scala.collection.Iterator.foreach$(Iterator.scala:941)
	at scala.collection.AbstractIterator.foreach(Iterator.scala:1429)
	at scala.collection.IterableLike.foreach(IterableLike.scala:74)
	at scala.collection.IterableLike.foreach$(IterableLike.scala:73)
	at scala.collection.AbstractIterable.foreach(Iterable.scala:56)
	at kafka.network.Processor.processCompletedReceives(SocketServer.scala:871)
	at kafka.network.Processor.run(SocketServer.scala:761)
	at java.base/java.lang.Thread.run(Thread.java:829)
[2022-06-17 05:19:58,969] ERROR Closing socket for 127.0.0.1:9092-127.0.0.1:50526-11 because of error (kafka.network.Processor)
org.apache.kafka.common.errors.InvalidRequestException: Error parsing request header. Our best guess of the apiKey is: 0
Caused by: java.nio.BufferUnderflowException
	at java.base/java.nio.Buffer.nextGetIndex(Buffer.java:651)
	at java.base/java.nio.HeapByteBuffer.getShort(HeapByteBuffer.java:350)
	at org.apache.kafka.common.protocol.ByteBufferAccessor.readShort(ByteBufferAccessor.java:38)
	at org.apache.kafka.common.message.RequestHeaderData.read(RequestHeaderData.java:119)
	at org.apache.kafka.common.message.RequestHeaderData.<init>(RequestHeaderData.java:83)
	at org.apache.kafka.common.requests.RequestHeader.parse(RequestHeader.java:93)
	at kafka.network.Processor.$anonfun$processCompletedReceives$1(SocketServer.scala:875)
	at kafka.network.Processor.$anonfun$processCompletedReceives$1$adapted(SocketServer.scala:871)
	at scala.collection.Iterator.foreach(Iterator.scala:941)
	at scala.collection.Iterator.foreach$(Iterator.scala:941)
	at scala.collection.AbstractIterator.foreach(Iterator.scala:1429)
	at scala.collection.IterableLike.foreach(IterableLike.scala:74)
	at scala.collection.IterableLike.foreach$(IterableLike.scala:73)
	at scala.collection.AbstractIterable.foreach(Iterable.scala:56)
	at kafka.network.Processor.processCompletedReceives(SocketServer.scala:871)
	at kafka.network.Processor.run(SocketServer.scala:761)
	at java.base/java.lang.Thread.run(Thread.java:829)
[2022-06-17 05:19:58,970] ERROR Exception while processing request from 127.0.0.1:9092-127.0.0.1:50526-11 (kafka.network.Processor)
org.apache.kafka.common.errors.InvalidRequestException: Error parsing request header. Our best guess of the apiKey is: 0
Caused by: java.nio.BufferUnderflowException
	at java.base/java.nio.Buffer.nextGetIndex(Buffer.java:651)
	at java.base/java.nio.HeapByteBuffer.getShort(HeapByteBuffer.java:350)
	at org.apache.kafka.common.protocol.ByteBufferAccessor.readShort(ByteBufferAccessor.java:38)
	at org.apache.kafka.common.message.RequestHeaderData.read(RequestHeaderData.java:119)
	at org.apache.kafka.common.message.RequestHeaderData.<init>(RequestHeaderData.java:83)
	at org.apache.kafka.common.requests.RequestHeader.parse(RequestHeader.java:93)
	at kafka.network.Processor.$anonfun$processCompletedReceives$1(SocketServer.scala:875)
	at kafka.network.Processor.$anonfun$processCompletedReceives$1$adapted(SocketServer.scala:871)
	at scala.collection.Iterator.foreach(Iterator.scala:941)
	at scala.collection.Iterator.foreach$(Iterator.scala:941)
	at scala.collection.AbstractIterator.foreach(Iterator.scala:1429)
	at scala.collection.IterableLike.foreach(IterableLike.scala:74)
	at scala.collection.IterableLike.foreach$(IterableLike.scala:73)
	at scala.collection.AbstractIterable.foreach(Iterable.scala:56)
	at kafka.network.Processor.processCompletedReceives(SocketServer.scala:871)
	at kafka.network.Processor.run(SocketServer.scala:761)
	at java.base/java.lang.Thread.run(Thread.java:829)
[2022-06-17 05:25:10,923] INFO [GroupCoordinator 0]: Member[group.instance.id None, member.id consumer-DepositTeam-2-0ecd7a34-1da2-4c66-9b31-2993bbd2e97a] in group DepositTeam has left, removing it from the group (kafka.coordinator.group.GroupCoordinator)
[2022-06-17 05:25:10,923] INFO [GroupCoordinator 0]: Preparing to rebalance group DepositTeam in state PreparingRebalance with old generation 3 (__consumer_offsets-45) (reason: removing member consumer-DepositTeam-2-0ecd7a34-1da2-4c66-9b31-2993bbd2e97a on LeaveGroup) (kafka.coordinator.group.GroupCoordinator)
[2022-06-17 05:25:10,923] INFO [GroupCoordinator 0]: Group DepositTeam with generation 4 is now empty (__consumer_offsets-45) (kafka.coordinator.group.GroupCoordinator)
[2022-06-17 05:25:38,941] INFO [GroupCoordinator 0]: Preparing to rebalance group DepositTeam in state PreparingRebalance with old generation 4 (__consumer_offsets-45) (reason: Adding new member consumer-DepositTeam-2-2104697d-33f9-44e0-8ee4-9bb3dcafd015 with group instance id None) (kafka.coordinator.group.GroupCoordinator)
[2022-06-17 05:25:38,942] INFO [GroupCoordinator 0]: Stabilized group DepositTeam generation 5 (__consumer_offsets-45) (kafka.coordinator.group.GroupCoordinator)
[2022-06-17 05:25:38,946] INFO [GroupCoordinator 0]: Assignment received from leader for group DepositTeam for generation 5 (kafka.coordinator.group.GroupCoordinator)
[2022-06-17 05:26:23,286] INFO [GroupCoordinator 0]: Member[group.instance.id None, member.id consumer-DepositTeam-2-2104697d-33f9-44e0-8ee4-9bb3dcafd015] in group DepositTeam has left, removing it from the group (kafka.coordinator.group.GroupCoordinator)
[2022-06-17 05:26:23,286] INFO [GroupCoordinator 0]: Preparing to rebalance group DepositTeam in state PreparingRebalance with old generation 5 (__consumer_offsets-45) (reason: removing member consumer-DepositTeam-2-2104697d-33f9-44e0-8ee4-9bb3dcafd015 on LeaveGroup) (kafka.coordinator.group.GroupCoordinator)
[2022-06-17 05:26:23,286] INFO [GroupCoordinator 0]: Group DepositTeam with generation 6 is now empty (__consumer_offsets-45) (kafka.coordinator.group.GroupCoordinator)
[2022-06-17 05:26:31,558] INFO [GroupCoordinator 0]: Preparing to rebalance group DepositTeam in state PreparingRebalance with old generation 6 (__consumer_offsets-45) (reason: Adding new member consumer-DepositTeam-2-3a980d7d-af7b-47d2-8520-3242934af707 with group instance id None) (kafka.coordinator.group.GroupCoordinator)
[2022-06-17 05:26:31,558] INFO [GroupCoordinator 0]: Stabilized group DepositTeam generation 7 (__consumer_offsets-45) (kafka.coordinator.group.GroupCoordinator)
[2022-06-17 05:26:31,562] INFO [GroupCoordinator 0]: Assignment received from leader for group DepositTeam for generation 7 (kafka.coordinator.group.GroupCoordinator)
[2022-06-17 05:28:41,618] INFO [GroupCoordinator 0]: Member[group.instance.id None, member.id consumer-DepositTeam-2-3a980d7d-af7b-47d2-8520-3242934af707] in group DepositTeam has left, removing it from the group (kafka.coordinator.group.GroupCoordinator)
[2022-06-17 05:28:41,618] INFO [GroupCoordinator 0]: Preparing to rebalance group DepositTeam in state PreparingRebalance with old generation 7 (__consumer_offsets-45) (reason: removing member consumer-DepositTeam-2-3a980d7d-af7b-47d2-8520-3242934af707 on LeaveGroup) (kafka.coordinator.group.GroupCoordinator)
[2022-06-17 05:28:41,618] INFO [GroupCoordinator 0]: Group DepositTeam with generation 8 is now empty (__consumer_offsets-45) (kafka.coordinator.group.GroupCoordinator)
[2022-06-17 05:28:51,858] INFO [GroupCoordinator 0]: Preparing to rebalance group DepositTeam in state PreparingRebalance with old generation 8 (__consumer_offsets-45) (reason: Adding new member consumer-DepositTeam-2-82eff66b-0c9d-4062-a201-76627e715b1b with group instance id None) (kafka.coordinator.group.GroupCoordinator)
[2022-06-17 05:28:51,858] INFO [GroupCoordinator 0]: Stabilized group DepositTeam generation 9 (__consumer_offsets-45) (kafka.coordinator.group.GroupCoordinator)
[2022-06-17 05:28:51,863] INFO [GroupCoordinator 0]: Assignment received from leader for group DepositTeam for generation 9 (kafka.coordinator.group.GroupCoordinator)
[2022-06-17 05:29:03,867] INFO [GroupMetadataManager brokerId=0] Removed 0 expired offsets in 0 milliseconds. (kafka.coordinator.group.GroupMetadataManager)
[2022-06-17 05:36:29,261] INFO [GroupCoordinator 0]: Member[group.instance.id None, member.id consumer-DepositTeam-2-82eff66b-0c9d-4062-a201-76627e715b1b] in group DepositTeam has left, removing it from the group (kafka.coordinator.group.GroupCoordinator)
[2022-06-17 05:36:29,261] INFO [GroupCoordinator 0]: Preparing to rebalance group DepositTeam in state PreparingRebalance with old generation 9 (__consumer_offsets-45) (reason: removing member consumer-DepositTeam-2-82eff66b-0c9d-4062-a201-76627e715b1b on LeaveGroup) (kafka.coordinator.group.GroupCoordinator)
[2022-06-17 05:36:29,261] INFO [GroupCoordinator 0]: Group DepositTeam with generation 10 is now empty (__consumer_offsets-45) (kafka.coordinator.group.GroupCoordinator)
[2022-06-17 05:36:37,948] INFO [GroupCoordinator 0]: Preparing to rebalance group DepositTeam in state PreparingRebalance with old generation 10 (__consumer_offsets-45) (reason: Adding new member consumer-DepositTeam-2-4c2ea9c1-c7ae-4d3b-99fe-2d9a9e968210 with group instance id None) (kafka.coordinator.group.GroupCoordinator)
[2022-06-17 05:36:37,948] INFO [GroupCoordinator 0]: Stabilized group DepositTeam generation 11 (__consumer_offsets-45) (kafka.coordinator.group.GroupCoordinator)
[2022-06-17 05:36:37,952] INFO [GroupCoordinator 0]: Assignment received from leader for group DepositTeam for generation 11 (kafka.coordinator.group.GroupCoordinator)
[2022-06-17 05:39:03,867] INFO [GroupMetadataManager brokerId=0] Removed 0 expired offsets in 1 milliseconds. (kafka.coordinator.group.GroupMetadataManager)
[2022-06-17 05:40:36,692] INFO [GroupCoordinator 0]: Member[group.instance.id None, member.id consumer-DepositTeam-2-4c2ea9c1-c7ae-4d3b-99fe-2d9a9e968210] in group DepositTeam has left, removing it from the group (kafka.coordinator.group.GroupCoordinator)
[2022-06-17 05:40:36,692] INFO [GroupCoordinator 0]: Preparing to rebalance group DepositTeam in state PreparingRebalance with old generation 11 (__consumer_offsets-45) (reason: removing member consumer-DepositTeam-2-4c2ea9c1-c7ae-4d3b-99fe-2d9a9e968210 on LeaveGroup) (kafka.coordinator.group.GroupCoordinator)
[2022-06-17 05:40:36,692] INFO [GroupCoordinator 0]: Group DepositTeam with generation 12 is now empty (__consumer_offsets-45) (kafka.coordinator.group.GroupCoordinator)
[2022-06-17 05:40:44,838] INFO [GroupCoordinator 0]: Preparing to rebalance group DepositTeam in state PreparingRebalance with old generation 12 (__consumer_offsets-45) (reason: Adding new member consumer-DepositTeam-2-ac352f9e-3e72-41d3-bf08-15cc46b52b5d with group instance id None) (kafka.coordinator.group.GroupCoordinator)
[2022-06-17 05:40:44,839] INFO [GroupCoordinator 0]: Stabilized group DepositTeam generation 13 (__consumer_offsets-45) (kafka.coordinator.group.GroupCoordinator)
[2022-06-17 05:40:44,843] INFO [GroupCoordinator 0]: Assignment received from leader for group DepositTeam for generation 13 (kafka.coordinator.group.GroupCoordinator)
[2022-06-17 05:49:03,867] INFO [GroupMetadataManager brokerId=0] Removed 0 expired offsets in 0 milliseconds. (kafka.coordinator.group.GroupMetadataManager)
[2022-06-17 05:59:03,867] INFO [GroupMetadataManager brokerId=0] Removed 0 expired offsets in 0 milliseconds. (kafka.coordinator.group.GroupMetadataManager)
